package.json

```diff
@@ -27,8 +27,8 @@
     "socket.io": "^1.7.2"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
-    "mocha": "^8.2.0",
-    "socket.io-client": "^2.3.1"
+    "chai": "4.2.0",
+    "mocha": "6.2.3",
+    "socket.io-client": "2.3.1"
```

Comment:
Can you just revert this back to previous version. This is acceptable on heroku.

Reply:
Done!